apps/design-system/src/types/index.ts

```diff
@@ -94,6 +94,14 @@ export enum OperationStatus {
   skipped = 'skipped',
 }
 
+export enum UploadAssetStatus {
```

Comment:
This should just be uploadStatus, it does not matter what type of file its uploading